cmd/controller/main.go

```diff
@@ -62,12 +62,20 @@ func main() {
 	flag.StringVar(&opts.Images.ImageDigestExporterImage, "imagedigest-exporter-image", "", "The container image containing our image digest exporter binary.")
 	flag.StringVar(&opts.Images.WorkingDirInitImage, "workingdirinit-image", "", "The container image containing our working dir init binary.")
 
+	flag.StringVar(&opts.SpireConfig.TrustDomain, "spire-trust-domain", "example.org", "Experimental: The SPIRE Trust domain to use.")
+	flag.StringVar(&opts.SpireConfig.SocketPath, "spire-socket-path", "unix:///spiffe-workload-api/spire-agent.sock", "Experimental: The SPIRE agent socket for SPIFFE workload API.")
+	flag.StringVar(&opts.SpireConfig.ServerAddr, "spire-server-addr", "spire-server.spire.svc.cluster.local:8081", "Experimental: The SPIRE server address for workload/node registration.")
+	flag.StringVar(&opts.SpireConfig.NodeAliasPrefix, "spire-node-alias-prefix", "/tekton-node/", "Experimental: The SPIRE node alias prefix to use.")
```

Comment:
These are not valid default values it seems, and need to be customised to be able to use this feature.
Is there any reason these cannot be loaded from a config map like other configuration parameters? 
Setting them here implies to me that these values should not be changed at runtime. If that is the case, I think it's fine to keep the configs in here. I would ask you then to include them in the controller manifest, perhaps commented out, so that if someone would like to use the feature they can simply uncomment in the manifests and set the correct value. I know it's in the `spire.md` docs but I think having it commented in the controller manifest would help.

Reply:
@afrittoli Thanks for the review comment. I think it should be possible to change these flags into a configmap. I am trying that out now. Will update once I am done.

Reply:
I moved the spire config to a config map. These changes are in a separate commit so we could probably get them out if not ok.
Pros of configmap
1. The config can be changed after pipelines controller starts. So if a spire server goes down and a new one is instantiated, the config map can be changed and Tekton will pick up the change.

BTW the above is also a con of the configmap as now the SPIRE server can be changed after the pipelines comes up.

@pxp928 @lumjjb @afrittoli PTAL and comment on the config map approach.